### ISSUE TITLE
Release Plans

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/bootstrap_lite#28]
Since this theme is still fairly unstable and officially in "beta" mode. Here is my thought, let's be relatively liberal in cutting new beta release while we try to get this theme stable.

I'm thinking that I'll issue a new release as often as weekly, as long as there are any changes. Hopefully, this will help us get more eyes on the code and more testing done. 

Anyone using on production should be cautious about implementing these updates. But if you are able, please do test them.  

I'm looking for input (atten @bugfolder) on when you think we should go ahead and call this stable.

(Also, I welcome feedback on this rough plan.)

**Comment by stpaultim** [comment]
NOTE: Normally, I assumed that most Backdrop projects don't cut an official release until they are stabile, because it's unusual to see a Beta or Alpha release on BackdropCMS.org.

But, we inherited this module with a Beta release on BackdropCMS.org AND there was some discussion in Zulip yesterday to indicate that maybe it's ok to have ALPHA or BETA release on BackdropCMS.org. 

Anyway, I am somewhat making this up as I go, which is why I do appreciate feedback on this issue (or complaints if someone thinks I'm doing this wrong).

**Comment by bugfolder** [comment]
Since I raised this question on Zulip, here's my thoughts after seeing the various answers (modulo my own biases and preconceptions of course).

First, I'm assuming that alpha and beta versions are (or should be) the same as prerelease status in Github; that is, if it's alpha or beta, it's marked prerelease.

Then, I think it's OK, even a good thing, to put prerelease modules on b.org, because people contemplating a migration are going to look there first to see if there's a B version of the D7 module they need. Even if it's prerelease, and they don't want to build a site from something prerelease, knowing that there's a prerelease version gives them hope that a release version might come soon, and they won't waste effort by trying to port their own version of the D7 module if there's a prerelease version already visibly in progress. (I think it's more likely that they'll do the module search on B.org versus looking through github/backdrop-contrib.)

So for all the modules I've ported, I pretty much put an alpha version on b.org as soon as flames didn't come shooting out when I turned it on. Fixed bugs as I found them and bumped the alpha number. But since I probably miss lots of corner cases in my own testing, I've waited till a few more users have tried it out with no reported issues before I've taken off the alpha/beta/prerelease setting.

So, I think it's good to have the beta version of this module on B.org. Finding it there in beta was what persuaded me to base a migration plan off of it, and to dive in and figure out what some of the remaining blocking issues were.
